### PR TITLE
This is the project Portfolio mobile version

### PR DESCRIPTION
**Pull request by ger619**
Create a new GitHub repo.

Create a new branch according to the[GitHub flow] i.e. created sidebar

Set up [a linter for HTML and CSS]

-Add descriptive README file.

-A html file with 3 interfaces i.e. landing page and about us

- A css file that includes all the files

- A test.md file

- Now you will build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. These sections of the mobile website should include:

- the "About myself" section (placed after the projects and before the contact form):

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  overflow: hidden;
+  box-shadow: 0 0.1rem 1rem rgba(0, 0, 0, 0.1);
+  border-radius: 1em;
+  background: #fff;
+}
+
```
Comment (AmaduKamara) [inline main.css]: [ Required Changes ]

- [ ] Please add bottom spacing on the cards. 
Your Design
image

The Figma Design
image

Hint: Add this bottom margin style to fix it. 
`margin-bottom: 88px;`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                </div>
+                <div class="card__footer">
+                    <div class="user">
+                        <div class="user__info">
+                            <ul>
+                                <li class="social-media-icons"><button type="button" class="btn">html</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+                            </ul>
+                            <h5><button type="button" class="btn-b">See Project</button></h5>
```
Comment (AmaduKamara) [inline index.html]: [ Required Changes ]
- [x] Please let the button stand alone by itself. You shouldn't place a button inside a heading tag. Please fix this

File: index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                </div>
+                <div class="card__footer">
+                    <div class="user">
+                        <div class="user__info">
+                            <ul>
+                                <li class="social-media-icons"><button type="button" class="btn">html</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+
+                            </ul>
```
Comment (AmaduKamara) [inline index.html]: [ Required Changes ]
- [ ] Please stick to just `ul` and `li` to list the items. It's not required to place a button inside of the `li` tag
- [ ] Also please adjust the styles of these badges to match the Figma design
You Design
image

The Figma Design
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻️♻️♻️

Hi @ger619,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF (2)

## Highlights
✔️ No linter errors
✔️ Preofeesiona README
✔️ Good PR description

## Required Changes ♻️

- [x] The headline section of your design is missing. Please add the headline section you did days ago to complete the look of your portfolio website.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (ger619) [reply index.html]: Done resolved

File: index.html (hunk 2 above)
Comment (ger619) [reply index.html]: Done.

File: main.css (hunk 1 above)
Comment (ger619) [reply main.css]: Done

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @ger619 👋🏻, 

Good job on making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — COMMENTED** [review]
Hi Abol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### I see you are repeating the week, therefore I will approve the project in order to not block you from moving forward. in the future when repeating a week please either create a new repo or reverse your project using git. I would advise you to create a new repo and copy all your existing code to that repo so you will start clean.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ger619** [comment]
Thanks @mikethreels , I need just need it to reflect on the portal so as to be able to make a request. Noted and thanks once again

**Pull request merged**